devicetypes/smartthings/ikea-button.src/ikea-button.groovy

```diff
@@ -188,7 +216,12 @@ def parse(String description) {
 		if ((description?.startsWith("catchall:")) || (description?.startsWith("read attr -"))) {
 			def descMap = zigbee.parseDescriptionAsMap(description)
 			if (descMap.clusterInt == zigbee.POWER_CONFIGURATION_CLUSTER && descMap.attrInt == 0x0021) {
-				event = getBatteryEvent(zigbee.convertHexToInt(descMap.value))
+				def batteryValue = zigbee.convertHexToInt(descMap.value)
+				if (isSomfySituo()) {
```

Comment:
Shouldn't need a manufacturer check here. This attribute reports the percentage in .5% increments so it should always be divided by 2. If there are devices that use this DTH that report the value incorrectly according to the spec then the manufacturer/model check should be for those devices.

Reply:
ok, since this dth only supported ikea devices (so far) - i reversed that condition.